### PR TITLE
Positioning-and-floating-element

### PR DESCRIPTION
**Pull request by ajkacca457**
**The new york times clone** 

- Created a header with a fixed navigation bar
-  Created a main element that contains multiple sections, images, and embedded videos from youtube.
- Created an aside element and positioned it on the right side with "position: absolute" property.
- Used grid to align-items from multiple sections
- Used grid-template-area property to create a layout of multiple sections. 
- Created a footer element that contains multiple unorder list.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,748 @@
...
+  <body>
+
+  <!--This is a header section and it includes nav -->
+
+    <header>
+      <nav class="nav flex">
+        <div class="first flex flex-prop">
+          <i class="fas fa-bars mr"></i>
+          <i class="fas fa-search mr"></i>
+          <a href="">Space & Cosmos</a>
```
Comment (SafaErden) [inline index.html]: - [x] Please consider using capital letters for this link to make it identical to the original page. (Space & Cosmos)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,748 @@
...
+        </div>
+
+        <div class="logo flex-prop">
+          <img src="images/nyt.png" alt="newyorktimes" class="nyt" />
```
Comment (SafaErden) [inline index.html]: - [x] The NY Times logo is a link (link to page), please add `<a>` tag for the NYT logo.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,748 @@
...
+      </section>
+
+      <section id="infaltion-section">
+        <h3 class="align-left">The Theory of Inflation</h3>
+        <p class="align-left caption">
+          Astronomers have found evidence to support the theory of inflation,
+          which explains how the universe expanded so uniformly and so quickly
+          in the instant after the Big Bang 13.8 billion years ago.
+        </p>
+        <img src="images/inflation-720.png" alt="inflation image" />
```
Comment (SafaErden) [inline index.html]: - [x] You shouldn't use screenshots of the original page as shortcut to bypass requirements. Please consider building the 'Theory of Inflation' section by using grid, flex or float.

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, your project is awesome 👍  it really looks exactly like the original. However, we need a little fix before we can proceed. Kindly find comments below:

- [x] The expandable footer should have a fixed position (It sticks to the bottom part of the screen, if you are logged in, you may not see it. So to view it you should log out.) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
image


- [x] Let us refactor our CSS file. Let us apply the DRY rule. Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

* no repeating of the same rule


<hr>

Kindly make these changes and submit another code review 
Happy coding_ 👨‍💻 
If you have any question, please contact me on Slack: @safaerden

File: index.html (hunk 3 above)
Comment (ajkacca457) [reply index.html]: Thank you for your helpful review. We worked on this and updated our project according to the required changes.

**Review by santiagorodriguezbermudez — APPROVED** [review]
# Project Approved 🥳🎉🎊

Nice job implementing the required changes! The page looks amazing. 

Moreover, kudos on using all of the original text. I know that takes a lot of time and patience. 🤓 

## Take into account the following before merging your PR🧐:

- You have spelling mistakes on your commits. This does not look professional. Kindly rebase your commits and change their names. Specifically, the last commit: "Added the inflation secion and styled with css grid"

- Every time you make changes to a code review, kindly write a descriptive comment that details what were the changes you implemented. This helps future TSE to check out what were the changes and improves your communication skills. 

Happy coding! 👨🏻‍💻
Cheers,
Santiago Rodriguez B.
Reach out on slack (@santiagorodriguezb) if you have any further questions / comments.

**Pull request merged**